lib/FundDistribution/FundDistributionView/FundDistributionView.css

```diff
@@ -0,0 +1,3 @@
+.fundDistributionAmount + .fundDistributionAmount:before {
+  content: ', ';
```

Comment:
IMHO, not the best way to do it. 
cc @NikitaSedyx @AndreiShumski2  any thoughts ?

Reply:
Added condition for comma

Reply:
Looks much better without `content:`